my-dsl.md

```diff
@@ -2,37 +2,46 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+I found the DSL [BUGS.](http://fisher.utstat.toronto.edu/pub/escobar/bug5/doc/manual05.ps)
 
```

Comment:
Just FYI, this link is a download for the language manual.  I wasn't bothered by it at all, but in the future it might be nice to have a download warning if the user is expecting to go to a webpage.